packages/@webex/plugin-meetings/test/unit/spec/meetings/index.js

```diff
@@ -1782,6 +1783,30 @@ describe('plugin-meetings', () => {
           it('creates the meeting from a rejected meeting info fetch and destroys it if failOnMissingMeetingInfo', async () => {
             checkCreateMeetingWithNoMeetingInfo(true, true);
           });
+
+          const checkCreateMeetingWithOne2OneType = async () => {
+            try {
+              const meeting = await webex.meetings.createMeeting(
+                'test destination',
+                _ONE_2_ONE_MEEETING_
+              );
+
+              assert.instanceOf(
+                meeting,
+                Meeting,
+                'createMeeting should eventually resolve to a Meeting Object'
+              );
+              assert.notCalled(webex.meetings.meetingInfo.fetchMeetingInfo);
+              assert.calledThrice(TriggerProxy.trigger);
+            } catch (err) {
+              assert.instanceOf(err, NoMeetingInfoError);
+            }
+          };
```

Comment:
I don't think we need a separate function block as we would use this method in only one test case. Let's add the logic directly to the callback of the test case in line 1807